hw7/3_exercise.py

```diff
@@ -0,0 +1,13 @@
+def mapper(func, *sequences):
+    if len(sequences)>1:
+        while True:
+            list.append(func(sequences[0][0],sequences[0][0],))
```

Comment:
what is sequences[0][0],sequences[0][0],
how do you know that your function will accept 2 parameters?